mesheryctl/internal/cli/root/system/start.go

```diff
@@ -311,8 +314,14 @@ func start() error {
 
 		checkFlag := 0 //flag to check
 
+		// Get the Docker configuration
+		dockerCfg, err := cliconfig.Load(dockerconfig.Dir())
+		if err != nil {
+			return errors.Wrap(err, utils.SystemError("failed to load config file"))
```

Comment:
@Azanul  will you have this turned into a MeshKit-style error, complete with probable cause, error code, and suggested remediation?

Reply:
Check it out